R/fa.R

```diff
@@ -159,77 +154,68 @@ fa <- function(name,
     get_length_value_unit(css_length = height)
     get_length_value_unit(css_length = width)
 
-    extra_attrs <- "preserveAspectRatio=\"none\" "
+    svg_attrs <- c(svg_attrs, list(preserveAspectRatio = "none"))
```

Comment:
```suggestion
    svg_attrs$preserveAspectRatio <- "none"
```

This is (a tiny bit) faster, but more importantly, it's clearer about types. I dislike the `x <- c(x, list(a=1))` syntax, because it can "magically" promote `x` to a list.

In the example below, `y` starts out as `list(x)`, but after the same `c()` operation, they end up with identical values:

```R
x <- 1
y <- list(x)

x <- c(x, list(b=2, c=3))

y <- c(y, list(b=2, c=3))

identical(x, y)
#> [1]  TRUE
```